monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+### _Triptych_
+
+Triptych a été conçu pour adresser ces défis: il s’agit d’un nouvel algorithme destiné aux signatures de cercles développé par Sarang Noether et Brandon Goodell des Laboratoires de Recherche de Monero (MRL) avec la participation de Arthur Blue (RandomRun). Avec Triptych, la taille de la signature de cercle croît de manière logarithmique plutôt que linéaire. La croissance logarithmique est très lente, et présente un choix judicieux qui permet d’éviter de faire confiance à des tierces parties (et par là, faire échec à l’anonymat comme pour Zcash). Le délai de vérification des opérations croît toujours linéairement avec le nombre de leurres avec Triptych, mais les vérifications peuvent-être regroupées et triées par l'entremise d’algorithmes spécialisés, accélérant le traitement.
+
+Triptych parvient à faire fi de plusieurs difficultés posées par l’usage des signatures de cercles avec Monero. Il permet notamment d’établir un lien: la technique des signatures de cercles de Monero doit bloquer la réutilisation de clés publiques légitimes afin de prévenir le double usage d’un TXO. Triptych y parvient en utilisant des étiquettes de liaison ("linking tags"), similaires aux images cryptographiques actuelles. Triptych est aussi compatible avec les transactions confidentielles, qui permettent d’obfusquer les volumes de Monero échangés. Des données spéciales (“clés d’engagement - "commitment keys"), permettent de valider toutes les opérations de mouvement de Monero sans dévoiler leurs montants. Ces clés d’engagement, avec les clés publiques et les étiquettes de liaison, sont les trois éléments qui donnent naissance au nom de Triptych, qui signifie "formé de trois parties".
+
+### _Tests et Alternatives_
+
+Triptych permet une augmentation quantifiable de la rapidité d'utilisation de Monero. Un de ses créateurs, Sarang Noether, a signalé que le code d’essai donnait une amélioration d’exécution significative. En bref, l’utilisation de 63 leurres sous Tryptych est 10% plus rapide que l’utilisation de 10 leurres avec CLSAG. Ces performances présentent une amélioration fondamentale pour Monero, tout en étant pratique à l'utilisation.
+
+Il reste cependant du travail à faire: une inspection rigoureuse d’un nouvel algorithme tel que Triptych est toujours nécessaire avant son inclusion dans le code source de Monero. Bien que l’algorithme de Triptych ait été soumis à l’appréciation de chercheurs pairs (et accepté pour présentation au séminaire ESORICS CBT 2020), Triptych pourrait encore bénéficier de modifications pour optimiser d'avantage sa rapidité d’exécution. Le temps consacré à l’amélioration de Triptych prend toutefois la place de celui qui servirait à découvrir des solutions alternatives.
```

Comment:
Original sentence:
Le temps consacré à l’amélioration de Triptych prend toutefois la place de celui qui servirait à découvrir des solutions alternatives.
Suggested sentence:
Le temps consacrable à l'amelioration de Triptych peut toutefois être dédié à la recherche de solutions alternatives. 

Reason: More accurate translation in my opinion

Reply:
Integrated.